apps/setting/models_provider/impl/gemini_model_provider/gemini_model_provider.py

```diff
@@ -42,14 +45,25 @@
               GeminiImage),
 ]
 
-
+model_stt_info_list = [
+    ModelInfo('gemini-1.5-flash', '最新的Gemini 1.5 Flash模型，随Google更新而更新',
+              ModelTypeConst.STT,
+              gemini_stt_model_credential,
+              GeminiSpeechToText),
+    ModelInfo('gemini-1.5-pro', '最新的Gemini 1.5 Flash模型，随Google更新而更新',
+              ModelTypeConst.STT,
+              gemini_stt_model_credential,
+              GeminiSpeechToText),
+]
 
 model_info_manage = (
     ModelInfoManage.builder()
     .append_model_info_list(model_info_list)
     .append_model_info_list(model_image_info_list)
+    .append_model_info_list(model_stt_info_list)
     .append_default_model_info(model_info_list[0])
     .append_default_model_info(model_image_info_list[0])
+    .append_default_model_info(model_stt_info_list[0])
     .build()
 )
 
```

Comment:
There are a few minor improvements and optimizations that can be made to enhance the code:

1. **Remove Trailing Commas**: You have trailing commas after some dictionary entries (`gemini_llm_model_credential`, `gemini_image_model_credential`). These should be removed for consistency.

2. **Consistent Spacing**: Ensure consistent spacing around operators (e.g., `=`) and after commas.

3. **Import Order**: It might be helpful to sort the imports alphabetically for better readability and maintainability.

4. **Variable Naming**: Use meaningful variable names like `stt_models` instead of `model_stt_info_list`.

Here's the revised version with these changes:

```python
@@ -8,7 +8,6 @@
 from setting.models_provider.impl.gemini_model_provider.model.stt import GeminiSpeechToText
 from smartdoc.conf import PROJECT_DIR

-gemini_stt_model_credential = GeminiSTTModelCredential()

-model_info_list = [
+stt_models = [
     ModelInfo('gemini-1.0-pro', '最新的Gemini 1.0 Pro模型，随Google更新而更新',
              ModelTypeConst.GENERAL_MODEL,
              None,
@@ -42,13 +41,18 @@
               GeminiImage),
 ]

+
+models_stt_list = [
+    ModelInfo('gemini-1.5-flash', '最新的Gemini 1.5 Flash模型，随Google更新而更新',
+              ModelTypeConst.STT,
+              gemini_stt_model_credential,
+              GeminiSpeechToText),
+]

+
 model_info_manage = (
     ModelInfoManage.builder()
         .append_model_info_list(model_info_list)
         .append_model_info_list(stt_models)
         .append_default_model_info(default_model)
         .build()
 )

```

These changes make the code cleaner and more readable.